giraffe/src/components/Icon.scss

```diff
@@ -29,6 +29,58 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 
+  &.more:before {
+    content: '\e974';
+  }
+  &.drag-to-expand-white:before {
+    content: '\e901';
+    color: #fff;
+  }
+  &.drag-to-expand:before {
+    content: '\e902';
+  }
+  &.shield-solid:before {
+    content: '\e903';
+  }
+  &.lock:before {
+    content: '\e900';
+  }
+  &.heart-solid:before {
+    content: '\e96f';
```

Comment:
This entire file came from Clockface, [[L20](https://github.com/influxdata/clockface/blob/master/src/Styles/icon.scss#L20)]